azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/ChatSDKWrapper.kt

```diff
@@ -73,6 +74,17 @@ internal class ChatSDKWrapper(
         return future
     }
 
+    override fun startEventNotifications(context: Context) {
+        if (startedEventNotifications) return
+        chatAsyncClient.startRealtimeNotifications(context) {}
```

Comment:
can we do something like? chatAsyncClient.startRealtimeNotifications(context) {
            if(it == null) {
                // set bool to ture
            } else {
                throw it
            }
        }

Reply:
Using context as instance variable in https://github.com/Azure/communication-ui-library-android/pull/455/commits/093c6e208e9d4ce907bb469abfc93e324bc083e8